src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -78,9 +78,6 @@ public static class WorkerRuntimeLanguageHelper
             {
                 switch (wr)
                 {
-                    case WorkerRuntime.powershell:
-                        workerToDisplayStrings[wr] = "powershell (preview)";
-                        break;
```

Comment:
I think we can get rid of the method, right?

Reply:
Hi @ahmelsayed, my original thought was to keep it as is, so next time we add a new preview language, we can just add a few lines and we are ready to go. However, you are right, we can remove this method and simplify the logic in ```InitAction.cs```.